### DISCUSSION TITLE
Encryption verification is failed on "subscribe" staging URL

### DISCUSSION THREAD

**Discussion by akn9097** [ONDC-Official/developer-docs#63] [Q&A]
Hello ONDC team,
While I'm using this URL (https://staging.registry.ondc.org/subscribe), I'm getting below error response,

![subscribe](https://github.com/ONDC-Official/developer-docs/assets/154229551/6efa3eaa-9879-4c0b-82de-ca1ce66483a6)

And on that another side I have developed "subscriber_id " endpoint and host on server  (Domain name) 
https://www.antworksmoney.com/on_subscribe/

![on_subscribee](https://github.com/ONDC-Official/developer-docs/assets/154229551/b0f7b930-cc50-40e2-bd4d-f51d38d63710)

**Comment by rasika-sarang** [comment]
@akn9097 How you fixed the issue ? Please let me know I am also facing the same issue.
